usaspending_api/download/filestreaming/download_generation.py

```diff
@@ -531,33 +561,49 @@ def _top_level_split(sql, splitter):
 
 def execute_psql(temp_sql_file_path, source_path, download_job):
     """Executes a single PSQL command within its own Subprocess"""
-    try:
-        log_time = time.perf_counter()
-        temp_env = os.environ.copy()
-        if download_job and not download_job.monthly_download:
-            # Since terminating the process isn't guaranteed to end the DB statement, add timeout to client connection
-            temp_env["PGOPTIONS"] = f"--statement-timeout={settings.DOWNLOAD_DB_TIMEOUT_IN_HOURS}h"
-
-        cat_command = subprocess.Popen(["cat", temp_sql_file_path], stdout=subprocess.PIPE)
-        subprocess.check_output(
-            ["psql", "-q", "-o", source_path, retrieve_db_string(), "-v", "ON_ERROR_STOP=1"],
-            stdin=cat_command.stdout,
-            stderr=subprocess.STDOUT,
-            env=temp_env,
-        )
+    download_sql = Path(temp_sql_file_path).read_text()
+    if download_sql.startswith("\\COPY"):
+        # Trace library parses the SQL, but cannot understand the psql-specific \COPY command. Use standard COPY here.
+        download_sql = download_sql[1:]
```

Comment:
Oh nice, I didn't realize just removing the leading slash would help DD parse the SQL

Reply:
Yes, but unfortunately because this sql is usually so long, it has a limit of how much sql it will store.